lib/index.js

```diff
@@ -16,7 +16,7 @@ const pendingOps = Symbol('pendingOps');
 const fileStreamDone = Symbol('fileStreamDone');
 const acceptVersion = Symbol('acceptVersion');
 
-const supportedVersion = /^[123]\./;
+const supportedVersion = /^[1234]\./;
```

Comment:
Technically, we can't claim support for version 3 and below without modifying the fixture files. We *could* do this by switching on the version while iterating over the test material. That said, if we received no complaints about this behavior under the "version 3" guidelines, then there's little reason to fix things now.

Reply:
Modifying the fixture in what way? The only difference between Test262 v3 and v4 is that v4 disallows certain content in files that Test262-Stream doesn't even open (`_FIXTURE.js`). The only reason Test262-Stream needs to be updated is because it will refuse to operate on Test262 files once Test262's package.json is updated to v4.